net-mail/getmail/getmail-6.03.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{7,8} )
+PYTHON_REQ_USE="ssl?"
+
+DISTUTILS_USE_SETUPTOOLS=no
+inherit distutils-r1
+
+DESCRIPTION="A mail retriever with reliable Maildir and mbox delivery"
+HOMEPAGE="http://www.getmail6.org/"
+SRC_URI="https://github.com/getmail6/getmail6/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~ppc-macos ~x86-macos ~x64-solaris"
+IUSE="ssl"
```

Comment:
This USE flag seems to have no function in this ebuild.

Reply:
I think the USE flag toggles a dependency on `dev-lang/python[ssl]` with `PYTHON_REQ_USE="ssl?"` in line 6, right?

Reply:
I have to agree with @juippis here. The SSL use flag does not seem to change how getmail is built. In fact by looking at the code we can see getmail implicitly expects SSL to be available and it cannot be selectively disabled.

So we should drop the `ssl` USE flag and on line 6 we just explicitly require SSL enabled Python.